python/feathub/processors/flink/table_builder/flink_table_builder.py

```diff
@@ -536,46 +535,40 @@ def _get_table_from_sliding_feature_view(
                         "SlidingWindowTransform."
                     )
                 transform = feature.transform
-                window_aggs = sliding_window_agg_map.setdefault(
-                    SlidingWindowDescriptor.from_sliding_window_transform(transform),
-                    [],
+                if sliding_window_descriptor is None:
+                    sliding_window_descriptor = (
+                        SlidingWindowDescriptor.from_sliding_window_transform(transform)
+                    )
+                elif (
+                    sliding_window_descriptor
+                    != SlidingWindowDescriptor.from_sliding_window_transform(transform)
+                ):
+                    raise FeathubException(
+                        "There can be only one sliding window descriptor for one "
+                        "SlidingFeatureView."
```

Comment:
Let's improve this exception message for Feathub users. For example, "a SlidingFeatureView cannot have different keys or step size".

Given that this requirement has existed before this PR, it should have been covered by existing exceptions. Could you please verify this? Perhaps the condition
```python
elif (
    sliding_window_descriptor
    != SlidingWindowDescriptor.from_sliding_window_transform(transform)
)
```
Would never happen.

Reply:
It should never reach here. I updated SlidingFeatureView to let it report the group-by keys and step size, which is more intuitive IMO.